### ISSUE TITLE
aten::fused_moving_avg_obs_fake_quant' to ONNX opset version 14 is not supported

### ISSUE THREAD

**Issue by seamoonlight-YBY** [pytorch/pytorch#108834]
### 🐛 The operator of aten does not support it. Should I use the version of torch 1.10? I observe that in the documentation of torch 1.9 version," fused_moving_avg_obs_fake_quant" is a kind of torchscript. Therefore, in order to use the version of torch 1.9, what adjustments are needed for my exported onnx code?

 # my code  
    import torch
    import torch.nn as nn
    from torch.quantization import QuantStub, DeQuantStub
    import torch.quantization
      
    class Net(nn.Module):
          def __init__(self):
              super(Net, self).__init__()
              self.conv1 = nn.Conv2d(3, 6, 5)
              self.bn1 = nn.BatchNorm2d(6)
              self.relu1 = nn.ReLU()
              self.quant = QuantStub()
              self.dequant = DeQuantStub()
          def forward(self, x):
              x = self.quant(x)
              x = self.relu1(self.bn1(self.conv1(x)))
              x = self.dequant(x)
              return x
          def fuse_model(self):
              torch.quantization.fuse_modules(self, ['conv1', 'bn1', 'relu1'], inplace=True)
    net = Net()
    print('===== Before fuse_model:')
    print(net)
    print('===== After fuse_model:')
    net.train()
    #net.fuse_model()
    print(net)
    print('===== Setting qconfig:')
    # Specify quantization configuration
    # Start with simple min/max range estimation and per-tensor quantization of weights
    # net.qconfig = torch.quantization.get_default_qat_qconfig('fbgemm')
    net.qconfig = torch.quantization.get_default_qat_qconfig('qnnpack')
    print(net.qconfig)
    print('===== After torch.quantization.prepare:')
    torch.quantization.prepare_qat(net, inplace=True)
    print(net)
    
    input_fp32 = torch.randn((1, 3, 48, 48))
    output_x = net(input_fp32)
    print(output_x.shape)
    torch.onnx.export(net, input_fp32, './tmp.onnx', export_params=True, opset_version=14, input_names=['input'], 
                     operator_export_type=torch.onnx.OperatorExportTypes.ONNX_FALLTHROUGH,
                     output_names=['output'],
                     keep_initializers_as_inputs=True ,
                     )

# bug
    ============= Diagnostic Run torch.onnx.export version 2.0.0+cu117 =============                                                                                                                                                                                                                                                                                
    verbose: False, log level: Level.ERROR                                                                                                                                                                                                                                                                                                                          
    ======================= 0 NONE 0 NOTE 0 WARNING 4 ERROR ========================                                                                                                                                                                                                                                                                                
    ERROR: missing-standard-symbolic-function                                                                                                                                                                                                                                                                                                                       
    =========================================                                                                                                                                                                                                                                                                                                                       
    Exporting the operator 'aten::fused_moving_avg_obs_fake_quant' to ONNX opset version 14 is not supported. Please feel free to request support or submit a pull request on PyTorch GitHub: https://github.com/pytorch/pytorch/issues.                                                                                                                            
    None                                                                                                                                                                                                                                                                                                                                                            
    <Set verbose=True to see more details>                                                                                                                                                                                                                                                                                                                          
                                                                                                                                                                                                                                                                                                                                                                    
    ERROR: missing-standard-symbolic-function                                                                                                                                                                                                                                                                                                                       
    =========================================                                                                                                                                                                                                                                                                                                                       
    Exporting the operator 'aten::fused_moving_avg_obs_fake_quant' to ONNX opset version 14 is not supported. Please feel free to request support or submit a pull request on PyTorch GitHub: https://github.com/pytorch/pytorch/issues.                                                                                                                            
    None                                                                                                                                                                                                                                                                                                                                                            
    <Set verbose=True to see more details>                                                                                                                                                                                                                                                                                                                          
                                                                                                                                                                                                                                                                                                                                                                    
    ERROR: missing-standard-symbolic-function                                                                                                                                                                                                                                                                                                                       
    =========================================                                                                                                                                                                                                                                                                                                                       
    Exporting the operator 'aten::fused_moving_avg_obs_fake_quant' to ONNX opset version 14 is not supported. Please feel free to request support or submit a pull request on PyTorch GitHub: https://github.com/pytorch/pytorch/issues.                                                                                                                            
    None                                                                                                                                                                                                                                                                                                                                                            
    <Set verbose=True to see more details>                                                                                                                                                                                                                                                                                                                          
                                                                                                                                                                                                                                                                                                                                                                    
    ERROR: missing-standard-symbolic-function                                                                                                                                                                                                                                                                                                                       
    =========================================                                                                                                                                                                                                                                                                                                                       
    Exporting the operator 'aten::fused_moving_avg_obs_fake_quant' to ONNX opset version 14 is not supported. Please feel free to request support or submit a pull request on PyTorch GitHub: https://github.com/pytorch/pytorch/issues.                                                                                                                            
    None
    <Set verbose=True to see more details>



### Versions

torch.__version__ = 2.0.0+cu117
```[tasklist]
### Tasks
```


cc @jerryzh168 @jianyuh @raghuramank100 @jamesr66a @vkuzo @jgong5 @Xia-Weiwen @leslie-fang-intel

**Comment by thiagocrepaldi** [comment]
This model did export (without any code changes) using latest PyTorch main branch. Here is the graph

<img width="332" alt="image" src="https://github.com/pytorch/pytorch/assets/5469809/cc980168-e0b5-4748-8f39-2c90d46a1ddb">

**Comment by seamoonlight-YBY** [comment]
> This model did export (without any code changes) using latest PyTorch main branch. Here is the graph
> 
> <img alt="image" width="332" src="https://user-images.githubusercontent.com/5469809/267415749-cc980168-e0b5-4748-8f39-2c90d46a1ddb.png">

Thank you! What is your PyTorch version?

**Comment by thiagocrepaldi** [comment]
I used latest `main` branch of github.com/pytorch/pytorch, but it should also work on Pytorch 2.1.0 RC

If you cannot compile from source, try this: https://dev-discuss.pytorch.org/t/pytorch-release-2-1-0-final-rc-is-available/1502

**Comment by seamoonlight-YBY** [comment]
> I used latest `main` branch of github.com/pytorch/pytorch, but it should also work on Pytorch 2.1.0 RC
> 
> If you cannot compile from source, try this: https://dev-discuss.pytorch.org/t/pytorch-release-2-1-0-final-rc-is-available/1502

Alright. Thank you!

**Comment by HanXuMartin** [comment]
@thiagocrepaldi Hi, I did converted the onnx model successfully, but met one error when doing inference:
ort_session = onnxruntime.InferenceSession(my_onnx_model_file)

and the error is:
onnxruntime.capi.onnxruntime_pybind11_state.Fail: [ONNXRuntimeError] : 1 : FAIL : Load model from models/multi-task-36-opset12-slim.onnx failed:Fatal error: org.pytorch.aten:fused_moving_avg_obs_fake_quant(-1) is not a registered function/op

any suggestions?
